recipes/forwarder.rb

```diff
@@ -68,6 +75,13 @@
   notifies :restart, 'service[logstash-forwarder]'
 end
 
+# put logstash user into group for cert access
+group 'logstash-certs' do
+  action :manage
+  append true
+  members [node['logstash_forwarder']['user']]
+end
```

Comment:
Actually, that's not being used any more! that was the first attempt to avoid duplicated certs and keep things neat, but it didn't work out.  The user needs to be the owner on the cert, Group members were access denied.  I had two others review to see if we could get around it but no dice. :(